src/main/java/eu/bbmri_eric/negotiator/service/MolgenisServiceImplementation.java

```diff
@@ -67,4 +74,71 @@ public Optional<MolgenisBiobank> findBiobankById(String id) {
       return Optional.empty();
     }
   }
+
+  @Override
+  public List<MolgenisBiobank> findAllBiobanks() {
+    try {
+      String response =
+          webClient
+              .get()
+              .uri("/api/v2/eu_bbmri_eric_biobanks?num=10000&q=withdrawn==false&attrs=id,name")
+              .retrieve()
+              .bodyToMono(String.class)
+              .block();
+      JsonObject jsonResponse = JsonParser.parseString(response).getAsJsonObject();
+      JsonArray items = jsonResponse.getAsJsonArray("items");
+      List<MolgenisBiobank> biobanks = new ArrayList<MolgenisBiobank>();
+
+      for (JsonElement e : items) {
+        JsonObject jsonBiobank = e.getAsJsonObject();
+        MolgenisBiobank biobank =
+            new MolgenisBiobank(
+                jsonBiobank.get("id").getAsString(),
+                jsonBiobank.get("name").getAsString(),
+                jsonBiobank.get("_href").getAsString());
+        biobanks.add(biobank);
+      }
+      return biobanks;
+    } catch (WebClientResponseException | WebClientRequestException e) {
+      log.warn(e.getMessage());
+      log.warn("Molgenis is not reachable!");
+      return Collections.emptyList();
+    }
+  }
+
+  //  @Override
+  //  public List<MolgenisCollection> findAllCollectionsByBiobankId(MolgenisBiobank biobank) {
+  //    try {
+  //      String response =
+  //          webClient
+  //              .get()
+  //              .uri(
+  //                  String.format(
+  //
+  // "/api/v2/eu_bbmri_eric_collections?q=biobank==%s&num=10000&attrs=id,name,description",
+  //                      biobank.getId()))
+  //              .retrieve()
+  //              .bodyToMono(String.class)
+  //              .block();
+  //      JsonObject jsonResponse = JsonParser.parseString(response).getAsJsonObject();
+  //      JsonArray items = jsonResponse.getAsJsonArray("items");
+  //      List<MolgenisCollection> collections = new ArrayList<MolgenisCollection>();
+  //
+  //      for (JsonElement e : items) {
+  //        JsonObject jsonCollection = e.getAsJsonObject();
+  //        MolgenisCollection collection =
+  //            new MolgenisCollection(
+  //                jsonCollection.get("id").getAsString(),
+  //                jsonCollection.get("name").getAsString(),
+  //                jsonCollection.get("description").getAsString(),
+  //                biobank);
+  //        collections.add(collection);
+  //      }
+  //      return collections;
+  //    } catch (WebClientResponseException | WebClientRequestException e) {
+  //      log.warn(e.getMessage());
+  //      log.warn("Molgenis is not reachable!");
+  //      return Collections.emptyList();
+  //    }
+  //  }
```

Comment:
pls remove

Reply:
fixed in fb760484e9b2dbec0c0c5b584decfbd60305ad4d